pkg/cmd/kafka/topic/consume/consume.go

```diff
@@ -0,0 +1,215 @@
+package consume
+
+import (
+	"errors"
+	"time"
+
+	kafkaflagutil "github.com/redhat-developer/app-services-cli/pkg/cmd/kafka/flagutil"
+
+	"strings"
+
+	"github.com/redhat-developer/app-services-cli/pkg/cmd/kafka/kafkacmdutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/cmdutil/flagutil"
+	"github.com/redhat-developer/app-services-cli/pkg/core/ioutil/dump"
+	"github.com/redhat-developer/app-services-cli/pkg/core/localize"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/connection"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/contextutil"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	kafkainstanceclient "github.com/redhat-developer/app-services-sdk-go/kafkainstance/apiv1internal/client"
+	"github.com/spf13/cobra"
+)
+
+const (
+	DefaultOffset    = 0
```

Comment:
-1 would be better. 0 is an valid offset number.
If -1 is there we should not pass filter value for offset

Reply:
Okay, I thought having the -1 would fall into what we said about defaults being specific non-valid values so we know they are not set.

Reply:
Filtering values are problem. Because if they are not specified they need to be not included.
I do not think logic with offset 0 passed all the time will be valid :)

Offset could be "" then we can parse number from that string. WDYT?